agoradatatools/etl/transform.py

```diff
@@ -182,146 +164,105 @@ def transform_rna_seq_data(datasets: dict, adjusted_p_value_threshold: int):
     return diff_exp_data
 
 
-def transform_network(datasets: dict):
-    gene_info = datasets['gene_info']
-    networks = datasets['networks']
-
-    gene_info.rename(columns={"symbol": "hgnc_symbol"}, inplace=True)
-    gene_info = gene_info[['ensembl_gene_id', 'hgnc_symbol']]
-    gene_info.drop_duplicates(inplace=True)
-
-    networks = networks[
-        networks['genea_ensembl_gene_id'].isin(gene_info['ensembl_gene_id']) &
-        networks['geneb_ensembl_gene_id'].isin(gene_info['ensembl_gene_id'])]
-
-    merged = pd.merge(left=networks,
-                      right=gene_info,
-                      left_on='genea_ensembl_gene_id',
-                      right_on='ensembl_gene_id',
-                      how='left')
-
-    merged = pd.merge(left=networks,
-                      right=gene_info,
-                      left_on='geneb_ensembl_gene_id',
-                      right_on='ensembl_gene_id',
-                      how='left')
-    merged = merged[['genea_ensembl_gene_id', 'geneb_ensembl_gene_id',
-                     'genea_external_gene_name', 'geneb_external_gene_name', 'brainregion']]
-
-    return merged
+def fix_alias_field(df: pd.DataFrame) -> pd.DataFrame:
+    '''
+    The alias field in gene_info gets populated with NaN as rows are added. 
+    These need to be converted to an emtpy array to be consistent with the
+    rest of the column's values. 
+    '''
+    df['alias'] = df.apply(
+        lambda row: row['alias'] if isinstance(row['alias'], np.ndarray) \
+            else np.ndarray(0, dtype=object), axis = 1)
+        
+    return df
 
 
-def transform_gene_metadata(datasets: dict, adjusted_p_value_threshold, protein_level_threshold):
+def transform_gene_info(datasets: dict, adjusted_p_value_threshold, protein_level_threshold):
     '''
     This function will perform transformations and incrementally create a dataset called gene_metadata.
-    Each dataset will be left_joined onto gene_info.
+    Each dataset will be left_joined onto gene_metadata.
     '''
-    gene_info = datasets['gene_info']
+    gene_metadata = datasets['gene_metadata']
```

Comment:
should the dataset we are building be called gene_info for consistency?

Reply:
Yes we can do that. This line specifically though, in the config file I have defined 'gene_metadata' as the feather file, so `datasets['gene_metadata']` refers to that. Previously it was called 'gene_info', which was confusing since the dataset being built was also called 'gene_info'. I can rename the data frame that is built after this line in this function to `gene_info` though.

Reply:
Data frame has been renamed to `gene_info` in the transform. (commit [0cc2ee404510ce5d4b43beaa758e12f330aa63e4](https://github.com/jaclynbeck-sage/agora-data-tools/commit/0cc2ee404510ce5d4b43beaa758e12f330aa63e4))